data/bedrock/latest/types.yml

```diff
@@ -2043,6 +2055,24 @@ SoundType: varint =>
    - vault_deactivate
    - hurt_reduced
    - wind_charge_burst
+   - armor_crack_wolf
+   - armor_break_wolf
+   - armor_repair_wolf
+   - mace_smash_air
+   - mace_smash_ground
+   - mace_smash_heavy_ground
```

Comment:
mace_smash_heavy_ground is after `OMINOUS_BOTTLE_END_USE` and before `OMINOUS_ITEM_SPAWNER_SPAWN_ITEM_BEGIN`